docker-compose.override-osx.yml

```
@@ -0,0 +1,33 @@
+# Rename this file into docker-compose.override.yml:
+# mv docker-compose.override-osx.yml docker-compose.override.yml
+
+version: '3.7'
+
+services:
+  php:
+    environment:
+      XDEBUG_CONFIG: 'client_host=docker.for.mac.localhost client_port=9003'
```

```suggestion
      XDEBUG_CONFIG: 'client_host=docker.for.mac.localhost'
```

Not necessary to add the port if you add the `9000` one in your IDE config